src/components/Article.jsx

```diff
@@ -1 +1,17 @@
-// export const Article = () => ();
+import React from 'react';
+
+const Article = ({ title, text, date }) => (
```

Comment:
```suggestion
const Article = ({ 
  title, 
  text, 
  date, 
}) => (
```

Reply:
![image](https://user-images.githubusercontent.com/96409563/225837399-d1db9502-1830-4c97-bc35-9f91f511953d.png)
Not fixed